libs/auth/scopes/src/lib/license-api.scope.ts

```diff
@@ -2,4 +2,5 @@ export enum LicenseApiScope {
   licensesVerify = '@island.is/licenses:verify',
   licensesFirearm = '@island.is/licenses:firearm',
   licensesDisability = '@island.is/licenses:disability',
+  licensesDriving = '@island.is/licenses:driving',
```

Comment:
Does this scope exist in all environments? As it is not provided via seed script as part of this pull request.

Reply:
i'll add this